src/clone.c

```diff
@@ -297,27 +385,53 @@ SYS_FUNC(clone3)
 	if (prefix_fun != tprint_value_changed_struct_begin)
 		tprint_struct_end();
 
+	if (show_namespace && tcp->u_rval >= 0) {
+		tcp->auxstr = get_namespace_auxstr(tcp->u_rval, (uint64_t)arg.flags);
```

Comment:
Same issue with `tcp->u_rval` as in case of `clone`.

Reply:
fixed locally.